diagnostics/conf/routes

```diff
@@ -11,7 +11,10 @@ GET        /counts.gif                      controllers.DiagnosticsController.an
 GET        /robots.txt                      controllers.Assets.at(path="/public", file="robots.txt")
 
 POST       /css                             controllers.DiagnosticsController.css
-OPTIONS    /css                             controllers.DiagnosticsController.cssOptions
+OPTIONS    /css                             controllers.DiagnosticsController.postOptions
+
+POST       /csp-report                      controllers.DiagnosticsController.csp
```

Comment:
i think it's fine to do it as a cors request, can just use beacon as the report url